Native~/Events.h

```diff
@@ -14,9 +14,11 @@ namespace NativeRenderingPlugin
     enum class EventID
```

Comment:
Discovered I need to use a base index provided by Unity using:

`int IUnityGraphics::ReserveEventIDRange(int count); // reserves 'count' event IDs. Plugins should use the result as a base index when issuing events back and forth to avoid event id clashes.`

Will fix in next PR.